Network/DNS/Types.hs

```diff
@@ -244,10 +245,13 @@ pattern CDNSKEY    = TYPE  60 -- RFC 7344
 -- | Child-To-Parent Synchronization (RFC7477)
 pattern CSYNC :: TYPE
 pattern CSYNC      = TYPE  62 -- RFC 7477
+-- | Zone transfer (RFC5936)
+pattern AXFR :: TYPE
+pattern AXFR       = TYPE 252 -- RFC 5936
 -- | A request for all records the server/cache has available
 pattern ANY :: TYPE
 pattern ANY        = TYPE 255
--- | A request for all records the server/cache has available
+-- | Certificate Authority Authorization
```

Comment:
Please add an RFC reference <https://tools.ietf.org/html/rfc6844> and use the correct name: "Certification Authority Authorization".

Reply:
Will do